### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by jezdez**
Travis CI has a new pricing model which places limits on open source.

* https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing
* https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works

Many projects are moving to GitHub Actions instead, including Jazzband projects:

* https://github.com/orgs/jazzband/projects/1
* https://github.com/jazzband-roadies/help/issues/206

This is based on https://github.com/jazzband/contextlib2/pull/26.

TODO:

* [x] @jezdez to add `JAZZBAND_RELEASE_KEY` to the repo secrets.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,35 +1,34 @@
 [tox]
 envlist =
-    py{36,37,38,39}-django{22,30,31,master}
+    py{36,37,38,39}-dj{21,22,30,31,master}
```
Comment (charettes) [inline tox.ini:3]: I think we dropped support for 2.1

```suggestion
    py{36,37,38,39}-dj{22,30,31,master}
```

**Review by charettes — APPROVED** [review]
Except for the 2.1 references this LGTM

File: tox.ini (hunk 1 above)
Comment (jezdez) [reply tox.ini]: Good catch!

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 5
+      matrix:
+        python-version: ['3.6', '3.7', '3.8']
```
Comment (hramezani) [inline .github/workflows/test.yml:12]:
```suggestion
        python-version: ['3.6', '3.7', '3.8', '3.9']
```

**Review by hramezani — APPROVED** [review]

**Comment by jezdez** [comment]
Thanks for the quick review, all! ❤️

**Comment by charettes** [comment]
Worked flawlessly https://github.com/jazzband/django-model-utils/runs/1470115542

**Pull request merged**